src/main/java/com/altissia/clickandrun/domain/spreadsheet/Workbook.java

```diff
@@ -0,0 +1,83 @@
+package com.altissia.clickandrun.domain.spreadsheet;
+
+
+import com.altissia.clickandrun.domain.spreadsheet.validation.SheetValidation;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * An  workbook.
+ * Contains the workbook definition and the actual data of the workbook.
+ */
+public abstract class Workbook {
+
+    protected boolean ignoringSuperfluousHeaders;
```

Comment:
You sure ? Because the standard getter method name is "is" + class member and I prefer "isIgnoringSuperfluousHeaders" to "isIgnoreSuperfluousHeaders".